packages/ui/src/primitives/button.tsx

```diff
@@ -81,13 +94,31 @@ export const Button = React.forwardRef(function Button(
       type={type}
       {...props}
     >
-      {busy && spinnerWhenBusy ? (
-        <Spinner className='shrink-0 text-[1.125rem]'>Loading…</Spinner>
+      {busy && intent === 'primary' ? (
```

Comment:
Seeing a slight change in positioning of the text when we go into busy state. Would this be handled in icon adjustments ticket or apart of this work?


https://github.com/user-attachments/assets/440c8d6b-a2a5-46e3-bf2d-f84a08d8189e

Reply:
Happy to fix as a part of this one! If you approve I'll get it fixed first thing tomorrow, but no pressure